api/api.go

```diff
@@ -94,6 +96,7 @@ func WithNativeElection(committee committee.Committee) Option {
 // Server provides api for user to query blockchain data
 type Server struct {
 	bc                blockchain.Blockchain
+	bs                blocksync.BlockSync
```

Comment:
i don't think API/Metadata is the right place for this change? see comment in iotexproject/iotex-proto#105.

if the request is to know the local chain's sync status/height, adding a handler like /height in probe.go might be a better choice.

Reply:
@raullenchai